docs/docs/getting-started/bare-metal.md

```diff
@@ -28,7 +28,70 @@ Follow Canonical's instructions on [setting up Intel TDX on Ubuntu 24.04](https:
 
 Increase the `user.max_inotify_instances` sysctl limit by adding `user.max_inotify_instances=8192` to `/etc/sysctl.d/99-sysctl.conf` and running `sysctl --system`.
 
-## K3s Setup
+## K3s setup
 
 1. Follow the [K3s setup instructions](https://docs.k3s.io/) to create a cluster.
 2. Install a block storage provider such as [Longhorn](https://docs.k3s.io/storage#setting-up-longhorn) and mark it as the default storage class.
+
+## Preparing a cluster for GPU usage
+
+:::warning
+Currently, Contrast only supports GPU workloads on SEV-SNP-based clusters.
+:::
+
+To enable GPU usage on a Contrast cluster, some conditions need to be fulfilled for *each cluster node* that should host GPU workloads:
+
+1. The IOMMU needs to be activated. This can be checked by running `ls /sys/kernel/iommu_groups`. If the output contains the group indices (`0`, `1`, ...), the IOMMU is supported on the host.
+   If it doesn't, `intel_iommu=on` needs to be added to the kernel command line.
```

Comment:
```suggestion
1. You must activate the IOMMU. You can check by running `ls /sys/kernel/iommu_groups`. If the output contains the group indices (`0`, `1`, ...), the IOMMU is supported on the host.
   Otherwise, add `intel_iommu=on` to the kernel command line.
```
prefer active voice